dartagnan/src/main/java/com/dat3m/dartagnan/program/event/arch/vulkan/VulkanRMW.java

```diff
@@ -0,0 +1,35 @@
+package com.dat3m.dartagnan.program.event.arch.vulkan;
+
+import com.dat3m.dartagnan.expression.Expression;
+import com.dat3m.dartagnan.program.Register;
+import com.dat3m.dartagnan.program.event.Tag;
+import com.dat3m.dartagnan.program.event.common.RMWXchgBase;
+import com.dat3m.dartagnan.program.event.visitors.EventVisitor;
+
+public class VulkanRMW extends RMWXchgBase {
+
+    public VulkanRMW(Register register, Expression address, Expression value, String mo) {
+        super(register, address, value, mo);
+        this.addTags(Tag.Vulkan.ATOM);
+    }
+
+    private VulkanRMW(VulkanRMW other) {
+        super(other);
+        this.addTags(Tag.Vulkan.ATOM);
```

Comment:
Adding the Tag is not necessary, because the copy constructor call `super(other)` should do it already.

Reply:
removed